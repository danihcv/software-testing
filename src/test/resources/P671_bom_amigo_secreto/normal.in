10
544
1000
325
111
459
213
738
69
73
741
