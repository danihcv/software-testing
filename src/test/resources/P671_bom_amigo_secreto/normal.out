82958214
878320716
2320477
324852642
70321864
808306982
452856785
551978127
759229837
667127729
